building/build-debs/homeworld-admin-tools/build-package.sh

```diff
@@ -15,4 +19,3 @@ source ../common/package-build-helpers.sh
 
```

Comment:
These changes should probably be stuck below sourcing of package-build-helpers.sh.

Reply:
Just wondering, why is this the best practice?

Reply:
Good question. It's because source is kind of like import, so you do it at the top of the file for the same reason you usually do your imports at the top of a python file: the set of imports informs many things about what the file does, so by putting them at the top, the outline of what the file is doing becomes clearer to the reader.